client/app/components/group/_storeList/storeList.controller.js

```diff
@@ -17,7 +17,6 @@ class StoreListController {
     if (changes.groupId && angular.isDefined(changes.groupId.currentValue)) {
       this.Store.listByGroupId(changes.groupId.currentValue).then((data) => {
         this.CurrentStores.set(data);
-        this.storeList = angular.copy(this.CurrentStores.list);
```

Comment:
Right, although I was just looking at the store list not updating, not the map bit. Why not just get the `CurrentStores` service in the store map rather than pass as binding?